src/app/profile/profile.component.html

```diff
@@ -8,12 +8,24 @@ <h2>About {{ userName }}</h2>
     </div>
     <div class="col-xs-12 col-md-9">
       <h2>Selling Items</h2>
-      <div>
-        <img *ngFor="let item of userItemsForSale" class="img-reponsive" src="{{ item.itemImageSrc }}">
+      <div fxLayout="row">
```

Comment:
I was checking out the API for flex-layout on github, and I think we should also add `fxLayout.xs="column"` on the `<div/>`.

This ensure us that the cards will be displayed in columns when viewed on mobile web, which is nice.

Here's how I did it locally:

```
    <div fxLayout="row" fxLayout.xs="column">
      <app-item-card
        fxFlex="25%;"
        fxFlex.xs="100%;"
        class="item-card"
        [ngStyle.xs]="{'margin-bottom.px': 10}"
        *ngFor="let item of userItemsForSale"
        [itemName]="item.itemName"
        [itemAbout]="item.itemAbout"
        [itemRating]="item.itemRating"
        [itemPrice]="item.itemPrice"
        [itemImageSrc]="item.itemImageSrc">
      </app-item-card>
    </div>
    <h2>Recently Viewed Items</h2>
    <div fxLayout="row" fxLayout.xs="column">
      <app-item-card
        fxFlex="25%;"
        fxFlex.xs="100%;"
        class="item-card"
        [ngStyle.xs]="{'margin-bottom.px': 10}"
        *ngFor="let item of userItemsViewed"
        [itemName]="item.itemName"
        [itemAbout]="item.itemAbout"
        [itemRating]="item.itemRating"
        [itemPrice]="item.itemPrice"
        [itemImageSrc]="item.itemImageSrc">
      </app-item-card>
    </div>
```

Let me know what you think.